### PR TITLE
feat(route): 19楼头条

### LINKED ISSUE #9959: 嘉兴19楼论坛首页rss
### 类型

社交媒体

### 网站地址

https://jiaxing.19lou.com/

### 网站描述

嘉兴19楼是嘉兴地方性论坛。

### 需要生成什么内容？

嘉兴19楼首页上，头条+标签为“城事”、“热议”的标题。

image

### 额外描述

谢谢！

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by nczitzk**
## 该 PR 相关 Issue / Involved issue

Close #9959

## 完整路由地址 / Example for the proposed route(s)

```routes
/19lou/jiaxing
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/19lou/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    const e = SetCookie.arguments.length;
+    const d = e > 2 ? g[2] : null;
+    const i = e > 3 ? g[3] : null;
+    const f = e > 4 ? g[4] : null;
+    const a = e > 5 ? g[5] : false;
+    if (d !== null && d >= 0) {
+        c.setTime(c.getTime() + d * 24 * 60 * 60 * 1000);
+    }
+    return b + '=' + encodeURI(h) + (d === null || d < 0 ? (d === -1 ? '; expires=-1' : '') : '; expires=' + c.toGMTString()) + (i === null ? '' : '; path=' + i) + (f === null ? '' : '; domain=' + f) + (a === true ? '; secure' : '');
+}
```
Comment (TonyRL) [inline lib/v2/19lou/index.js:19]:
```suggestion
function SetCookie(cookieName, cookieValue) {
    const expires = new Date();
    const g = SetCookie.arguments;
    const e = SetCookie.arguments.length;
    const day = e > 2 ? g[2] : null;
    const path = e > 3 ? g[3] : null;
    const domain = e > 4 ? g[4] : null;
    const secure = e > 5 ? g[5] : false;
    if (day !== null && day >= 0) {
        expires.setTime(expires.getTime() + day * 24 * 60 * 60 * 1000);
    }
    return (
        cookieName +
        '=' +
        encodeURI(cookieValue) +
        (day === null || day < 0 ? (day === -1 ? '; expires=-1' : '') : '; expires=' + expires.toGMTString()) +
        (path === null ? '' : '; path=' + path) +
        (domain === null ? '' : '; domain=' + domain) +
        (secure === true ? '; secure' : '')
    );
}
```
More readability?

---

```js
        /**
         * Set cookie
         * @ param
         * @ return
         */
        setCookie: function(cookieName, cookieValue, seconds, path, domain, secure) {
            var expires = null;
            if (seconds != -1) {
                expires = new Date();
                expires.setTime(expires.getTime() + seconds);
            }
            document.cookie = [
                escape(cookieName),
                '=',
                escape(cookieValue),
                (expires ? '; expires=' + expires.toGMTString() : ''),
                (path ? '; path=' + path : '/'),
                (domain ? '; domain=' + domain : ''),
                (secure ? '; secure' : '')
            ].join("");
        },
```
Their `setCookie` function also escape `cookieName`.

Comment (nczitzk) [reply lib/v2/19lou/index.js:19]: The snippet came from the source code of the website and I just copied it 😅. Thanks for reviewing. Your practice is much better.

Comment (TonyRL) [reply lib/v2/19lou/index.js]: The above code is taken from
https://js2.citysbs.com/0.8.10.76/forum/App.component.js
I think you looked at the minified one: 
https://js2.citysbs.com/0.8.10.76/forum/AM.min.js

Comment (nczitzk) [reply lib/v2/19lou/index.js]: You caught me 😄.

**Pull request merged**